examples/DocQA/docker/run_RAG.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+echo "DOC_PATH=$(jq -r '.document_path' ./RAG_service.json)" > .env
```

Comment:
why do we need a .json and an .env? why can't we just use .env and avoid this manipulation completely?

Reply:
Thanks, this has been changed